FirebaseABTesting.podspec

```diff
@@ -24,6 +24,7 @@ Firebase Cloud Messaging and Firebase Remote Config in your app.
   s.ios.deployment_target = '10.0'
```

Comment:
nit: using Ruby variables for the platform target versions is more robust. See https://github.com/firebase/firebase-ios-sdk/blob/master/FirebaseCore.podspec#L21

Reply:
Done. Also updated remote config.